server.js

```diff
@@ -1,29 +1,70 @@
 import express from "express";
 import cors from "cors";
+import avocadoData from "./data/avocado-sales.json"
+import expressListEndpoints from 'express-list-endpoints';
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
+const port = process.env.PORT || 9000;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.json(endpoints)
+})
```

Comment:
when you list the endpoints you can add more info to document your API, for example you can filter the data by queries like `region` or `date`, you could add something like this:

```javascript
app.get("/", (req, res) => {
  const endpoints = expressListEndpoints(app);

  // Update documentation for the /avocados endpoint
  const avocadosEndpoint = endpoints.find(endpoint => endpoint.path === '/avocados');
  if (avocadosEndpoint) {
    avocadosEndpoint.queryParams = {
      region: "Filter by region (case-insensitive).",
      date: "Filter by date."
    };
  }

  res.json(endpoints);
});
``` 

this way you keep the usage of your API well documented ⭐